test/webapp-strategy-test.js

```diff
@@ -582,7 +612,10 @@ describe("/lib/strategies/webapp-strategy", function () {
 					done(e);
 				}
 			};
-			
+			webAppStrategy.fail = function (err) {
```

Comment:
You've changed the behavior of these tests. They should not fail... One option to keep your current test without affecting these is instead of checking for the absence of the `sub` field in the verify callback, look for a custom field that you add in your test, e.g.
```		
req.session[WebAppStrategy.AUTH_CONTEXT] = {
	identityTokenPayload: {'shouldFailVerifyCB':true}
};
```

Reply:
And once you make that change please delete these `webAppStrategy.fail`s that you added to other tests